buildSrc/src/main/java/Dependencies.kt

```diff
@@ -54,7 +54,7 @@ object Versions {
         const val test = "1.3.0"
         const val test_ext = "1.1.2"
         const val espresso = "3.3.0"
-        const val room = "2.3.0"
+        const val room = "2.4.0-alpha03"
```

Comment:
I see there's already a stable 2.4.1 version on https://developer.android.com/jetpack/androidx/releases/room. We probably don't want to use an alpha version of 2.4.0 since there's also a stable 2.4.0 version as well.

Reply:
> I see there's already a stable 2.4.1 version on https://developer.android.com/jetpack/androidx/releases/room. We probably don't want to use an alpha version of 2.4.0 since there's also a stable 2.4.0 version as well.

Thanks for the quick review! I actually tried that at first but unfortunately it fails with this error:

```
> Task :feature-addons:kaptGenerateStubsDebugKotlin FAILED
w: Runtime JAR files in the classpath should have the same version. These files were found in the classpath:
    /Users/skhamis/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk8/1.5.31/ff5d99aecd328872494e8921b72bf6e3af97af3e/kotlin-stdlib-jdk8-1.5.31.jar (version 1.5)
    /Users/skhamis/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-jdk7/1.5.31/77e0f2568912e45d26c31fd417a332458508acdf/kotlin-stdlib-jdk7-1.5.31.jar (version 1.5)
    /Users/skhamis/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.6.10/b8af3fe6f1ca88526914929add63cf5e7c5049af/kotlin-stdlib-1.6.10.jar (version 1.6)
    /Users/skhamis/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.6.10/c118700e3a33c8a0d9adc920e9dec0831171925/kotlin-stdlib-common-1.6.10.jar (version 1.6)
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath
e: warnings found and -Werror specified

> Task :compose-tabstray:compileDebugKotlin
```

My guess is because 2.4.1 upgraded to using kotlin 1.6 based on the commits in the 2.4.1 release
https://android.googlesource.com/platform/frameworks/support/+log/11c93b38ca49928eecf09ce48d7e6909a7264e05..3dda6c958a641fad4196ebc9b2e58a6f381ffdb4/room

which is a bit unfortunate but didn't want to make this a bigger change until a-c was ready to move to that version!